modules/test/playwright/tests/cookies-banner-web/cookiesBannerScript.spec.ts

```diff
@@ -0,0 +1,224 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+// @ts-ignore
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {loginTest} from '../../fixtures/loginTest';
+import getRandomString from '../../utils/getRandomString';
+
+export const test = mergeTests(loginTest());
+
+test('LPD-25701 Cookie Banner Script', async ({page}) => {
+
+	// Create Site
+
+	await page.getByLabel('Open Applications MenuCtrl+Alt+A').click();
+
+	await page.getByRole('tab', {name: 'Control Panel'}).click();
+
+	await page.getByRole('menuitem', {name: 'Sites'}).click();
+
+	await page.getByRole('link', {name: 'Add Site'}).click();
+
+	await page
+		.getByRole('button', {name: 'Select Template: Blank Site'})
+		.click();
+
+	const siteName = getRandomString();
+
+	const addSiteName = await page
+		.frameLocator('iframe[title="Add Site"]')
+		.getByLabel('Name\n\n\t\t\t\n\t\t\t\t\n\n\t\t\t\tRequired');
```

Comment:
This is a bit awkward. Maybe we can use [.and()](https://playwright.dev/docs/locators#matching-two-locators-simultaneously) here, checking `Name` and `Required`?

Reply:
No longer needed.